static-instrumentation/agent-instrumenter/build.gradle

```diff
@@ -0,0 +1,12 @@
+plugins {
+    id("otel.java-conventions")
+}
+
+ext.libraryName = 'OpenTelemetry Java Contrib Static Instrumentation Agent'
+description = 'OpenTelemetry Java Static Instrumentation Agent '
+
+jar {
+    manifest {
+        attributes('Main-Class': 'io.opentelemetry.javaagent.StaticInstrumenter')
+    }
+}
```

Comment:
I can't add a comment on the .gitkeep files. Please change to `src/test/java` (and expect to write the tests with our usual junit + assertj + mockito + awaitility)